MetaGenerator/TimeKeeper.cs

```diff
@@ -59,18 +59,14 @@ private void PrintMeasurements()
         }
 
         private long TotalTime()
-        {
-            return _measurements.Select(m => m.ElapsedMilliseconds).Sum();
-        }
+            => _measurements
+            .Select(m => m.ElapsedMilliseconds).Sum();
 
         private int EventNameColumnWidth()
-        {
-            return _measurements.Aggregate(0, (max, m) => Max(max, m.EventName.Length)) + 1;
-        }
+            => _measurements
+            .Aggregate(0, (max, m) => Max(max, m.EventName.Length)) + 1;
```

Comment:
indentation

Reply:
Addressed in 9fc64c88dcd906d2d0de07d7ad6281413165b994